project-base/app/src/Model/Store/StoreDataFactory.php

```diff
@@ -46,7 +54,10 @@ public function createFromStore(Store $store): StoreData
         $storeData->city = $store->getCity();
         $storeData->postcode = $store->getPostcode();
         $storeData->country = $store->getCountry();
-        $storeData->openingHours = $store->getOpeningHours();
+        $storeData->openingHours = array_map(
+            fn (OpeningHours $openingHours): OpeningHoursData => $this->openingHourDataFactory->createFromOpeningHour($openingHours),
+            $store->getOpeningHours(),
+        );
```

Comment:
I would probably move this method into the factory, but I don't mind it this way.